easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.54.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.54.0'
+
+homepage = 'https://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs%s/' % version.replace('.', ''),
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['0646bb97f6f4d10a763f4919c54fa28b4fbdd3dff8e7de3410431c81762cade0']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libpng', '1.6.37'),
+    ('freetype', '2.10.4'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('expat', '2.2.9'),
+    ('GLib', '2.68.2'),
+    ('cairo', '1.16.0'),
+    ('LibTIFF', '4.2.0'),
+]
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore toolchain
+    ('binutils', '2.36.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+# Do not use local copies of zlib, jpeg, freetype, and png
+preconfigopts = "mv zlib zlib.no && mv jpeg jpeg.no && mv freetype freetype.no && mv libpng libpng.no && "
+preconfigopts += 'export LIBS="$LIBS -L$EBROOTZLIB/lib -lz" && '
+
+configopts = "--with-system-libtiff --enable-dynamic"
+
+parallel = 1
```

Comment:
@branfosj I had no issues running this build with 4 cores. See https://gist.github.com/f0ad62fd86c26a711d51c19c0a94fcbd

Reply:
I saw the issue when using 8 cores, so I've switched to `maxparallel = 4` in 176c485.